tests/system/providers/google/cloud/cloud_build/example_cloud_build_trigger.py

```diff
@@ -0,0 +1,152 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG that displays interactions with Google Cloud Build.
+This DAG relies on the following OS environment variables:
+* PROJECT_ID - Google Cloud Project to use for the Cloud Function.
+"""
+
+import os
+from datetime import datetime
+from typing import Any, Dict
+
+from airflow import models
+from airflow.models.baseoperator import chain
+from airflow.providers.google.cloud.operators.cloud_build import (
+    CloudBuildCreateBuildTriggerOperator,
+    CloudBuildDeleteBuildTriggerOperator,
+    CloudBuildGetBuildTriggerOperator,
+    CloudBuildListBuildTriggersOperator,
+    CloudBuildRunBuildTriggerOperator,
+    CloudBuildUpdateBuildTriggerOperator,
+)
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+
+DAG_ID = "example_gcp_cloud_build_trigger"
+
+GCP_SOURCE_REPOSITORY_NAME = "test-cloud-build-repository"
+
+# [START howto_operator_gcp_create_build_trigger_body]
+create_build_trigger_body = {
+    "name": "test-cloud-build-trigger",
+    "trigger_template": {
+        "project_id": PROJECT_ID,
+        "repo_name": GCP_SOURCE_REPOSITORY_NAME,
+        "branch_name": "main",
+    },
+    "filename": "example_cloud_build.yaml",
+}
+# [END howto_operator_gcp_create_build_trigger_body]
+
+# [START howto_operator_gcp_update_build_trigger_body]
+update_build_trigger_body = {
+    "name": "test-cloud-build-trigger",
+    "trigger_template": {
+        "project_id": PROJECT_ID,
+        "repo_name": GCP_SOURCE_REPOSITORY_NAME,
+        "branch_name": "master",
+    },
+    "filename": "example_cloud_build.yaml",
+}
+# [END START howto_operator_gcp_update_build_trigger_body]
+
+# [START howto_operator_create_build_from_repo_body]
+create_build_from_repo_body: Dict[str, Any] = {
+    "source": {"repo_source": {"repo_name": GCP_SOURCE_REPOSITORY_NAME, "branch_name": "master"}},
+    "steps": [{"name": "ubuntu", "args": ['echo', 'Hello world']}],
+}
+# [END howto_operator_create_build_from_repo_body]
+
+
+with models.DAG(
+    DAG_ID,
+    schedule_interval='@once',
```

Comment:
```suggestion
    schedule='@once',
```
Same here. See #25410 and #25648.